### DISCUSSION TITLE
FR Add name, state option to new Badges

### DISCUSSION THREAD

**Discussion by Mariusthvdb** [home-assistant/frontend#21523] [Dashboard] [answered]
seems a straightforward ask not requiring a lot of explanation. 
Given the fact icon is not optional in any of the current 3 options available

<img width="468" alt="Scherm­afbeelding 2024-08-01 om 09 37 20" src="https://github.com/user-attachments/assets/5e9b5fb8-e67d-4759-acdc-46a3543ebcf1">

and setting an empty icon with `icon: []` does this:

<img width="89" alt="Scherm­afbeelding 2024-08-01 om 09 30 06" src="https://github.com/user-attachments/assets/93193477-8b60-4599-9a98-ef45fa3e3a47">

please consider **adding an extra option without icon,** to be able to do this:

<img width="153" alt="Scherm­afbeelding 2024-08-01 om 09 42 38" src="https://github.com/user-attachments/assets/bf06ef82-cafe-4c1e-a449-ce98e69df67a">

the right badge is a custom badge but not very straightforward and requiring templating:
```
type: custom:mushroom-template-badge
content: '{{state_translated(entity)}}'
icon: ''
color: ''
entity: person.marijn
label: Marijn
```

Thanks for considering

**Comment by c2bull** [comment]
Please please give consideration to this. I immediately rolled back because the new badges with the forced icons ruin the top row experience for me. They may take up less vertical space than the previous badges but they take so much more horizontal that it breaks my flow of skimming the badges for the information I need.

**Comment by piitaya** [comment, marked as answer]
The display option has be changed to 3 booleans in Home Assistant 2024.9 for more flexibility : 
- show_icon (Boolean)
- show_name (Boolean)
- show_state (Boolean)

**Reply by Mariusthvdb** [reply]
thank you, tested and working beautifully!
